### ISSUE TITLE
[Bug]: Contents link missing aria roles

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shinyGovstyle#186]
### Overview

The contents_links() functions that give the left hand navigation are missing ARIA roles that tell other technology what kind of component they are.

### Steps to Reproduce

Inspect the HTML for the showcase app. The div with id "govuk-contents-box" is the one where we'd expect an ARIA label and / or role.

### Expected Behavior

We need to research whether these are set as navigation or tabset (technically they are tabs, but I think navigation might be more fitting given it's a separate links list and they're not tabs in the traditional way for keyboard users).

**Comment by cjrace** [comment]
Closing this as not planned. Since v0.2.0 (and discussion in https://github.com/dfe-analytical-services/shinyGovstyle/pull/176), we've now deprecated the contents links in favour of the horizontal nav.
